code/modules/migrations/SS13_Prefs/021-antag_objectives.dm

```diff
@@ -0,0 +1,13 @@
+/datum/migration/sqlite/ss13_prefs/_021
+	id = 21
+	name = "Toggle Solo Antag Objectives"
+
+/datum/migration/sqlite/ss13_prefs/_021/up()
+	if(!hasColumn("client","antag_objectives"))
+		return execute("ALTER TABLE `client` ADD COLUMN antag_objectives INTEGER DEFAULT 1")
```

Comment:
This makes people have objectives by default
While I'm not against the PR in and off itself, it really makes the poll worthless

Reply:
yeah set this to 0 as well

Reply:
Yeah, my bad. Worked fine locally, so I happily forgot to update the file that actually makes it work on the server as well.